File: app/src/main/java/com/example/tutorial/CircleActivity.java
package com.example.tutorial;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class CircleActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new MovementView(this));
    }
}

File: app/src/main/java/com/cruelgeroi/apphub/activities/CircleActivity.kt
package com.cruelgeroi.apphub.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.cruelgeroi.apphub.ui.MovementView

class CircleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(MovementView(this))
    }
}